js/react_native/android/build.gradle

```diff
@@ -20,6 +20,25 @@ def getExtOrIntegerDefault(name) {
   return rootProject.ext.has(name) ? rootProject.ext.get(name) : (project.properties['OnnxruntimeModule_' + name]).toInteger()
 }
 
+def getOnnxruntimePackageName() {
+  // locate user's project dir
+  def reactnativeRootDir = project.rootDir.parentFile
+  // get package.json file in root directory
+  def packageJsonFile = new File(reactnativeRootDir, 'package.json')
+  // read field 'ortPackageName'
```

Comment:
maybe there's a better name than `ortPackageName`. the package name is actually 'onnxruntime-react-native' right?

what if we look for an option `onnxruntimeExtensionsEnabled` to be set? or if you want a general config option, could put a whole subobject in `onnxruntimeOptions`.

Reply:
for now, I made it a field just named as `onnxruntimeExtensionsEnabled` with bool type. in the future if we want more options for the project in the future, I can probably make it as an option type.